src/PointwiseFunctions/Hydro/MassWeightedFluidItems.hpp

```diff
@@ -7,7 +7,9 @@
 
 #include "DataStructures/DataBox/Tag.hpp"
 #include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Structure/ObjectLabel.hpp"
 #include "Evolution/Systems/GrMhd/ValenciaDivClean/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "ParallelAlgorithms/Events/Tags.hpp"
```

Comment:
I think we can't depend the `hydro` lib on `ParallelAlgorithms/Events`. For now just template `MassWeightedCoordsCompute` on the two coord tags.